Sources/Utils/Pages/PagesController.swift

```diff
@@ -38,13 +38,29 @@ class PagesController: UIViewController {
   override func viewDidLayoutSubviews() {
     super.viewDidLayoutSubviews()
 
-    if scrollView.frame.size.width > 0 {
-      once.run {
-        goAndNotify()
+    guard scrollView.frame.size.width > 0 else {
+        return
+    }
+
+    once.run {
+      DispatchQueue.main.async {
+        self.scrollToAndSelect(index: self.selectedIndex, animated: false)
       }
+
+      notify()
     }
   }
 
+  override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
```

Comment:
Actually, the design of this repo was to support portrait only

Reply:
well - my fixes are feature now) our testers checked it out. All fixes are working 👍